gpiozero/boards.py

```diff
@@ -1156,6 +1156,8 @@ def close(self):
         if self._master:
             m = self._master
             self._master = None
+            self._socket = None
+            self._value = None
```

Comment:
Hmmm, perhaps the `value` property (and also `socket`?) should call `_check_open` ? https://github.com/RPi-Distro/python-gpiozero/blob/master/gpiozero/devices.py#L217
(just like `__repr__` does)